numba/tests/test_parfors.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function, division, absolute_import
+
+import gc
+
+import numpy as np
+
+from numba import njit, vectorize
+from numba import unittest_support as unittest
+from numba import compiler, typing, typeof, ir, utils
+from numba.compiler import Pipeline, _PipelineManager, Flags
+from numba.targets import cpu
+#from .support import MemoryLeakMixin, TestCase
+
+
+class TestParfors(unittest.TestCase):
+
+    def test_arraymap(self):
+        @njit(parallel=True)
+        def axy(a, x, y):
+            return a * x + y
+
+        A = np.linspace(0,1,10)
+        X = np.linspace(2,1,10)
+        Y = np.linspace(1,2,10)
+
+        output = axy(A,X,Y)
+        expected = A*X+Y
+        np.testing.assert_array_equal(expected, output)
```

Comment:
Thanks for the suggestion. Done. Moving forward, we need similar functionality for the Numba IR level (inspect_IR?).